fvtest/jitbuildertest/WorklistTest.cpp

```diff
@@ -0,0 +1,277 @@
+/*******************************************************************************
+ *
+ * (c) Copyright IBM Corp. 2017, 2017
+ *
+ *  This program and the accompanying materials are made available
+ *  under the terms of the Eclipse Public License v1.0 and
+ *  Apache License v2.0 which accompanies this distribution.
+ *
+ *      The Eclipse Public License is available at
+ *      http://www.eclipse.org/legal/epl-v10.html
+ *
+ *      The Apache License v2.0 is available at
+ *      http://www.opensource.org/licenses/apache2.0.php
+ *
+ * Contributors:
+ *    Multiple authors (IBM Corp.) - initial implementation and documentation
+ ******************************************************************************/
+
+#include "JBTestUtil.hpp"
+
+#include "ilgen/BytecodeBuilder.hpp"
+#include "ilgen/VirtualMachineState.hpp"
+
+#define EXPECT_BICEQ(expected_index, actual_index) do {\
+   EXPECT_EQ((expected_index), (actual_index)) << "Unexpected bytecode index (likely implies bytecode was not found in worklist)."; \
```

Comment:
I think the indices not being equal implies the worklist either missed or added a bytecode it should not have. "not found in worklist" implies some kind of search which I think is misleading in this use case.